ui/dashboards/src/context/DashboardProvider/DashboardProvider.tsx

```diff
@@ -95,33 +95,45 @@ function initStore(props: DashboardProviderProps) {
   } = props;
 
   const {
-    spec: { display, layouts, panels, duration },
+    spec: { display, duration },
     metadata,
   } = dashboardResource;
+
+  let {
+    spec: { layouts, panels },
+  } = dashboardResource;
+
+  // Set fallbacks
```

Comment:
It might be worth adding a little clarification about the generic (i.e. not chronosphere-specific) "why" for this for future contributors. I could see someone with less context being slightly confused because I think the perses Go backend prevents you from having these values be `null`.